### PR TITLE
Update readme with gigalixir workarounds

### PR DESCRIPTION
**Pull request by rhnonose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,15 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
...
-   6. `gigalixir migrate $APP_NAME`
+   
+   6. `MIX_ENV=prod mix release --env=prod`
+      `MIX_ENV=prod DATABASE_URL="$YOUR_DB_URL" mix ecto.migrate`
 
+      `$YOUR_DB_URL` is in the format: `ecto://user:pass@host:port/db` and you can find those parameters by running `gigalixir databases $APP_NAME`
 
+      This workaround is necessary because the app won't start without running the migrations.
+   
+   7. Connect to the console with `gigalixir remote_console $APP_NAME` and manually run the seeds.
```
Comment (ggpasqualino) [inline README.md]: what's the command to run the seeds there?

Comment (ghatighorias) [reply README.md]: Seeds should be run manually no ?

File: README.md [hunk 2]
```diff
@@ -35,7 +35,15 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
...
    4. `gigalixig set_config $APP_NAME DATABASE_URL $YOUR_DB_URL`
+      
+      If you don't have a database, you can create one by running `gigalixir create_database $APP_NAME` and waiting for it to become `AVAILABLE`.
    5. `git push gigalixir master`
-   6. `gigalixir migrate $APP_NAME`
+   
+   6. `MIX_ENV=prod mix release --env=prod`
+      `MIX_ENV=prod DATABASE_URL="$YOUR_DB_URL" mix ecto.migrate`
 
+      `$YOUR_DB_URL` is in the format: `ecto://user:pass@host:port/db` and you can find those parameters by running `gigalixir databases $APP_NAME`
```
Comment (ghatighorias) [inline README.md]: Or alternatively using this command 
`gigalixir configs $APP_NAME`

File: README.md (hunk 1 above)
Comment (ggpasqualino) [reply README.md]: yes, so?

Comment (ghatighorias) [reply README.md]: so don't you think after connecting to the server via console they can run the seeds?
well i assume it doesn't hurt to mention it either

Comment (ggpasqualino) [reply README.md]: > well i assume it doesn't hurt to mention it either
sorry that's a must for me, if I am not developer in this project and just want to spin it up, that will be a hassle

which command would you run anyway? as far as I know mix is not available in the remote console

Comment (rhnonose) [reply README.md]: Updated with detailed instructions.

File: README.md [hunk 3]
```diff
@@ -35,7 +35,18 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
...
+   
+   6. `MIX_ENV=prod mix release --env=prod`
+      `MIX_ENV=prod DATABASE_URL="$YOUR_DB_URL" mix ecto.migrate`
+
+      `$YOUR_DB_URL` is in the format: `ecto://user:pass@host:port/db` and you can find those parameters by running `gigalixir databases $APP_NAME`
+
+      This workaround is necessary because the app won't start without running the migrations.
+   
+   
+   7. Copy the contents of the `priv/repo/seeds.exs` file
```
Comment (ggpasqualino) [inline README.md]: are sure this works? seeds is an exs file, the remote console is an erlang console

**Comment by ggpasqualino** [comment]
there are lots of unrelated changes in this pr, maybe the merge from master didn't go well.
could you clean it up?

File: README.md (hunk 3 above)
Comment (rhnonose) [reply README.md]: Yes, `gigalixir`'s remote console is `iex` (I believe it connects through a different node but not sure).

File: README.md [hunk 4]
```diff
@@ -35,7 +35,18 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
...
    3. `APP_NAME=$(gigalixir create --name course-planner)`
    4. `gigalixig set_config $APP_NAME DATABASE_URL $YOUR_DB_URL`
+      
+      If you don't have a database, you can create one by running `gigalixir create_database $APP_NAME` and waiting for it to become `AVAILABLE`.
    5. `git push gigalixir master`
-   6. `gigalixir migrate $APP_NAME`
-
-
+   
+   6. `MIX_ENV=prod mix release --env=prod`
```
Comment (ggpasqualino) [inline README.md]: why is `MIX_ENV=prod mix release --env=prod` necessary? it doesn't seem like the local release is used later

Comment (rhnonose) [reply README.md]: I'm not sure, it's mentioned in gigalixir's workaround step.
Maybe it checks if the release is being built correctly?

**Comment by ggpasqualino** [comment]
@rhnonose ping
> there are lots of unrelated changes in this pr, maybe the merge from master didn't go well.
could you clean it up?

**Comment by rhnonose** [comment]
@ggpasqualino  Removed the merge issues.

**Pull request merged**